sydent/db/hashing_metadata.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class HashingMetadataStore:
+    def __init__(self, sydent):
+        self.sydent = sydent
+
+    def retrieve_value(self, name):
+        """Return a value from the hashing_metadata table
+        
+        :param name: The name of the db column to return the value for
+        :type name: str
+
+        :returns a value corresponding to the specified name, or None if a
+        value does not exist
+        """
+        cur = self.sydent.db.cursor()
+        res = cur.execute("select %s from hashing_metadata" % name)
+        row = res.fetchone()
+
+        if not row:
+            return None
+        return row[0]
+
+    def store_values(self, names_and_values):
```

Comment:
Given this is only used once I think it'd be better just to have a `store_pepper` function. Not least because it avoids the possibility that someone passes a user defined `name` and we end up with an SQL injection attack.